spec/helpers/sword_helper_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+class DummyClass
+  include SwordHelper
+end
+
+RSpec.describe SwordHelper do
+  describe 'collection_info_for_service_document' do
+    it 'contains expected values' do
+      dc = DummyClass.new
+      info = dc.collection_info_for_service_document('test-pq')
+      expect(info).to have_key(:slug)
+    end
+  end
+
+  describe 'pull_credentials' do
+    it 'returns array with Basic auth info' do
+      dc = DummyClass.new
+      encoded_auth = 'Basic Zmlyc3R0ZXN0ZGVwb3NpdG9yOmZpcnN0ZGVwb3NpdG9ycGFzc3dk'
+      request_double = instance_double(ActionDispatch::Request)
+      allow(request_double).to receive(:headers).and_return({'Authorization' => encoded_auth})
+      depositor, passwd = dc.pull_credentials request_double
+      expect(depositor).to eq('firsttestdepositor')
+      expect(passwd).to eq('firstdepositorpasswd')
+    end
+  end
+
+  describe 'create_deposit' do
+    it 'returns service document content' do
+      dc = DummyClass.new
+      deposit = create_deposit('test_user',
+                               'test_collection_slug',
+                               ['a.txt','b.txt'],
+                               'The test title',
+                               'ac:12345',
+                               ['ac:54321', 'ac:0984'],
+                               true,
+                               'tmp/somewhere/')
+      expect(deposit).to be_an_instance_of(Deposit)
+      end
+  end
+
+  describe 'service_document_content' do
+    context 'with config info' do
+      it 'returns service document content' do
+        dc = DummyClass.new
+        expect(dc.service_document_content).to include 'sword_version'
+      end
+    end
+  end
+
+  describe 'service_document_xml' do
+    context 'with mime type and package type specified' do
+      it 'returns' do
```

Comment:
are the it descriptions here and on line 73 supposed to specify what is returned?